config/config.go

```diff
@@ -30,14 +31,13 @@ import (
 	"github.com/grafana/regexp"
 	"github.com/prometheus/common/config"
 	"github.com/prometheus/common/model"
-	"github.com/prometheus/sigv4"
-	"gopkg.in/yaml.v2"
-
 	"github.com/prometheus/prometheus/discovery"
 	"github.com/prometheus/prometheus/model/labels"
 	"github.com/prometheus/prometheus/model/relabel"
 	"github.com/prometheus/prometheus/storage/remote/azuread"
 	"github.com/prometheus/prometheus/storage/remote/googleiam"
+	"github.com/prometheus/sigv4"
+	"gopkg.in/yaml.v2"
```

Comment:
These are supposed to be in the other group as they are not part of this repository.
So just un-do this part of the change.